com.ibm.wala.scandroid/src/main/java/org/scandroid/util/AndroidAnalysisContext.java

```diff
@@ -150,36 +148,6 @@ public static SSAPropagationCallGraphBuilder makeVanillaZeroOneCFABuilder(
         ZeroXInstanceKeys.ALLOCATIONS | ZeroXInstanceKeys.CONSTANT_SPECIFIC);
   }
 
-  /**
```

Comment:
Deleting since this method just calls itself in an infinite recursion.  Clearly no other code has been calling it